src/main/java/com/google/cloud/tools/appengine/cloudsdk/AppEngineComponentsNotInstalledException.java

```diff
@@ -0,0 +1,11 @@
+package com.google.cloud.tools.appengine.cloudsdk;
+
+import com.google.cloud.tools.appengine.api.AppEngineException;
+
+public class AppEngineComponentsNotInstalledException extends AppEngineException {
```

Comment:
added Javadoc. This seems close enough to AppEngineException to justify subclassing. I can change to a different exception if you like, but that would require clients to update.